tests/unit/test_auth_client.py

```diff
@@ -0,0 +1,230 @@
+import globus_sdk
+from tests.framework import (CapturedIOTestCase,
+                             get_client_data, get_user_data,
+                             SDKTESTER1A_NATIVE1_AUTH_RT)
+from globus_sdk.auth import (GlobusNativeAppFlowManager,
+                             GlobusAuthorizationCodeFlowManager)
+from globus_sdk.exc import GlobusAPIError
+
+
+class AuthClientTests(CapturedIOTestCase):
+
+    def setUp(self):
+        """
+        Instantiates an AuthClient with an access_token authorizer
+        """
+        super(AuthClientTests, self).setUp()
+
+        self.access_token = self.test_oauth2_token()
+        self.ac = globus_sdk.AuthClient(
+            authorizer=globus_sdk.AccessTokenAuthorizer(self.access_token),
+            client_id=get_client_data()["native_app_client1"]["id"])
+
+    def test_get_identities(self):
```

Comment:
This test is quite long, and I think it would be beneficial to break it up a little bit.
e.x. If `go@globusid.org` has changed and returns different data, we should still correctly test that sending `usernames` and `ids` results in an error.

We don't necessarily need to be very strict about where/how we break things down. Under the most rigorous regimen, we'd have a distinct test for each behavior of each call, with each set of parameters we give it -- seems a bit like overkill, IMHO.

However, I think it's useful, especially with a test like this which is vulnerable to breakage due to external changes, to separate testing of error conditions from working features.

I'd also be curious about @corpulentcoffee's and @jaswilli's opinions on this matter -- where and how they like to have unit tests broken apart.